swagger_documentation/Makefile

```diff
@@ -0,0 +1,133 @@
+configure:
```

Comment:
@pdan93  could we also have a `make clean`  that removes build-ed html and clients and an `make clients` that builds  all clients